adapters/stroeerCore/stroeercore.go

```diff
@@ -0,0 +1,114 @@
+package stroeerCore
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type StroeerCoreAdapter struct {
```

Comment:
This should be named `adapter`. There is no need to duplicate the package name and there is no need to export the type by starting with an uppercase letter.